src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/phase/modes/PhaseIntave.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2025 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.features.module.modules.exploit.phase.modes
+
+import net.ccbluex.liquidbounce.config.types.Choice
+import net.ccbluex.liquidbounce.config.types.ChoiceConfigurable
+import net.ccbluex.liquidbounce.event.tickHandler
+import net.ccbluex.liquidbounce.features.module.modules.exploit.phase.ModulePhase
+import net.minecraft.network.packet.c2s.play.PlayerActionC2SPacket
+import net.minecraft.util.math.Direction
+import kotlin.math.cos
+import kotlin.math.sin
+
+/**
+ * @anticheat Intave
+ * @anticheatVersion 14.9.0
+ * @testedOn mineblaze.net, go.mineberry.org
+ * @author larryngton
+ * Only works on 1.8.x
+ */
+object PhaseIntave : Choice("Intave") {
```

Comment:
```suggestion
internal object PhaseIntave : Choice("Intave") {
```

& move the `PhaseHypixel` & `PhaseIntave` to parent folder `phase`